recipes/xz_utils/all/conanfile.py

```diff
@@ -50,6 +50,13 @@ def _effective_msbuild_type(self):
     @property
     def _msbuild_target(self):
         return "liblzma_dll" if self.options.shared else "liblzma"
+    
+    @property
+    def _use_msbuild(self):
+        assume_clang_cl = (self.settings.os == "Windows"
+                           and self.settings.compiler == "clang"
+                           and self.settings.get_safe("compiler.runtime") is not None)
+        return is_msvc(self) or assume_clang_cl
```

Comment:
Can't it be simplified with just `self.settings.get_safe("compiler.runtime") is not None`?